src/main/java/se/kth/sda/skeleton/comments/Comment.java

```diff
@@ -46,6 +46,11 @@
 
     LocalDateTime createdTime;
 
+    @Lob
+    private byte[] fileByte;
+    private String fileType;
+    private String fileName;
```

Comment:
Could we use the same field names as for Post:
    
```
    private byte[] image;
    private String mediaType;
    private String videoName;
```

Reply:
The fields has been rename as Post